app/controllers/media_controller.rb

```diff
@@ -649,7 +649,7 @@ def search_results
           end
         end
       end
-      sort = params[:project] == "keks" ? "Quiz" : params[:project]&.capitalize
+      sort = params[:project] == "quiz" ? "Quiz" : params[:project]&.capitalize
```

Comment:
The else-statement of the ternary should always work, also for `quiz` right?

Reply:
Thanks, fixed.